### PR TITLE
Delta3 ui bottom rebased

### PR DESCRIPTION
**Pull request by soyarsauce**
### What this PR does

* Update Difftool BottomPanel UI to clearer "area filter" and date pickers
* Update Difftool BottomPanel to load into Terria's BottomDock
* Rearrange MapButton layout in DOM to properly reflow with BottomDock

Addresses https://github.com/TerriaJS/terriajs/issues/4371 https://github.com/TerriaJS/terriajs/issues/4372 & https://github.com/TerriaJS/terriajs/issues/4375

### Checklist

-   [x] unit tests n/a
-   [x] I've updated CHANGES.md with what I changed.

### CODE REVIEW

**Comment by reginapramesti** [comment]
image
Viewing the map on my laptop screen cuts off parts of the delta panel, so maybe we need to add a scrollbar as a temporary quick fix? And also the "No map data enabled" part covers the delta panel too in smaller screens 😬

**Review by reginapramesti — CHANGES_REQUESTED** [review]
Looks really good 👏  I think we just need to make the delta panel more visible on slightly smaller screens (since it doesn't take too small a screen to make it difficult to read/use). Should be good to go once we give that a quick fix! @soyarsauce

File: lib/ReactViews/Workbench/Controls/ViewingControls.jsx [hunk 1]
```diff
@@ -195,6 +195,9 @@ const ViewingControls = observer(
     },
 
     openDiffTool() {
+      // Disable timeline
+      // Should we do this? Difference is quite a specific use case
+      this.props.item.terria.timelineStack.removeAll();
```
[the remark this answers is not in the document]
Comment (soyarsauce) [reply lib/ReactViews/Workbench/Controls/ViewingControls.jsx:200]: Yeah if they explicitly always want timeline, then we leave it so

**Comment by soyarsauce** [comment]
> Looks really good 👏 I think we just need to make the delta panel more visible on slightly smaller screens (since it doesn't take too small a screen to make it difficult to read/use). Should be good to go once we give that a quick fix! @soyarsauce

Delta panel now resizes against viewport + some buffer, it will have to do for now.

**Comment by soyarsauce** [comment]
https://github.com/TerriaJS/terriajs/pull/4533/commits/d3e9c0b61915ee6327c5d4bb08ffb3a3f21ce393 also fixes https://github.com/TerriaJS/terriajs/issues/4537

**Review by reginapramesti — APPROVED** [review]
Thanks heaps for making it work on smaller screen 👏 really appreciate it! Looks good to me now

**Pull request merged**